src/components/bootcamps/BootcampsGrid.tsx

```diff
@@ -0,0 +1,170 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import React, { useState } from 'react'
+import { DataGrid, GridColumns, GridRowId } from '@mui/x-data-grid'
+import { useTranslation } from 'next-i18next'
+import { useBootcampsList } from 'common/hooks/bootcamps'
+import router from 'next/router'
+import InfoIcon from '@mui/icons-material/Info'
+import EditIcon from '@mui/icons-material/Edit'
+import { AlertStore } from 'stores/AlertStore'
+import { deleteBootcamp } from 'common/rest'
+import { useMutation } from 'react-query'
+import HighlightOffSharpIcon from '@mui/icons-material/HighlightOffSharp'
+import { Box, Button, ButtonGroup, Container, Stack, Typography } from '@mui/material'
+import DeleteModal from './layout/DeleteModal'
+import BootcampModal from './layout/BootcampModal'
+import Layout from './layout/Layout'
+import { pink } from '@mui/material/colors'
+import { green } from '@mui/material/colors'
+import Link from 'next/link'
+import DeleteSelectedModal from './layout/DeleteSelectedModal'
+import { BootcampResponse } from 'gql/bootcamps'
+import { routes } from 'common/routes'
+
+export default function BootcampsGrid() {
+  const { data = [] } = useBootcampsList()
+  const [open, setOpen] = useState(false)
+  const [details, setDetails] = useState(null || {})
+  const [deleteData, setDeleteData] = useState(null || {})
```

Comment:
@Yakimov1337 Why do we have this double default here - `null || {}`

This check will always lead to `{}` as `null` is always `false`

It will be the same to write it as 

```suggestion
  const [details, setDetails] = useState({})
  const [deleteData, setDeleteData] = useState({})
```

Right?